operators/hpe-csi-operator/deploy/crds/storage.hpe.com_hpecsidrivers_crd.yaml

```diff
@@ -44,6 +44,17 @@ spec:
             imagePullPolicy:
               description: Image Pull Policy for HPE CSI driver images
               type: string
+            iscsi:
+              description: Iscsi parameters to be configured
+              properties:
+                chapPassword:
+                  type: string
+                chapUser:
+                  type: string
+              required:
+              - chapPassword
```

Comment:
we can remove them as required params. Automatic CRD update will cause operator to kill CSI driver running with old CR.

Reply:
sure